Front-end/parquick/src/components/Messenger/Messenger.jsx

```diff
@@ -83,10 +122,10 @@ function Messenger(): React.MixedElement {
                     <div className="chat-menu-container">
                         <h3 className='conversations-title'>Conversations</h3>
                         {conversations.map((conv) => {
-                            return <div onClick={() => setCurrentChat(conv)} key={conv._id} >
+                            return <div onClick={() => setCurrentConversation(conv)} key={conv._id} >
                                 <Conversation
                                     conversation={conv}
-                                    isSelected={currentChat?._id && currentChat?._id === conv._id}
+                                    isSelected={currentConversation?._id && currentConversation?._id === conv._id}
```

Comment:
is there a specific reason we check `currentConversation._id` for, I guess existence, and then check again for equality with `conv._id`?